bok_choy/page_object.py

```diff
@@ -173,6 +187,7 @@ def __init__(self, browser):
             PageObject
         """
         self.browser = browser
+        self.verify_accessibility = bool(os.environ.get('VERIFY_ACCESSIBILITY'))
```

Comment:
What happens if I `export VERIFY_ACCESSIBILITY=False` or `export VERIFY_ACCESSIBILITY=0`?   I think that would result in self.verify_accessibility being True.